--- GithubSlackBot/github_bot.py	2018-02-18 17:35:04.000000000 +0900
+++ GithubSlackBot/github_bot.py	2018-02-18 17:36:10.000000000 +0900
@@ -100,14 +100,3 @@
                 'type': 'message',
                 'text': 'User removed'
             }))
-
-token = os.environ['TOKEN']
-get_url = requests.get('https://slack.com/api/rtm.connect?token=' + token)
-print(get_url.json()['url'])
-socket_endpoint = get_url.json()['url']
-print('Connecting to', socket_endpoint)
-
-websocket.enableTrace(True)
-ws = websocket.WebSocketApp(socket_endpoint, on_message=message)
-
-ws.run_forever()
\ No newline at end of file
